package/cluster/definition.yaml

```diff
@@ -30,6 +30,46 @@ spec:
                 type: object
                 description: Cluster configuration parameters.
                 properties:
+                  deletionPolicy:
+                    type: string
+                    default: Delete
+                    description: Delete or Orphan
+                    enum:
+                      - Delete
+                      - Orphan
+                  region:
+                    type: string
+                    default: us-west-2
+                    description: Amazon AWS region
+                    enum:
```

Comment:
I'm not sure we should have all the regions available (including Gov regions) unless we have verified that the Composition can work in that environment.

Reply:
Removed Gov regions.